### PR TITLE
Added the kubernetes takeover

### PR DESCRIPTION
**Pull request by inuwa**
## Done

Added Improving Productivity with kubernetes takeover
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Refresh the homepage until you see the "Improving Productivity with Kubernetes" takeover shows up Ensure it matches the [copy doc](https://docs.google.com/document/d/16YVNq8GST-yD7B3jhSwQ7_aZ0fc7Ly7l5MD4TQAhAXQ/edit?ts=5bd9ac7e) and [design spec](https://github.com/ubuntudesign/web-squad/issues/883)

## Issue / Card

Fixes https://github.com/ubuntudesign/web-squad/issues/884

### CODE REVIEW

File: templates/takeovers/_improving_productivity_kubernetes.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<section class="p-strip--image is-dark is-deep u-image-position js-takeover p-improving-prod-takeover">
+      <div class="row">
+        <div class="col-6" >
+          <h1 style="font-weight: 100;" class="u-no-margin--bottom u-sv3">Improving productivity with Ubuntu Kubernetes</h1>
+          <p class="p-heading--four">Canonical helped investment company, Allan Gray, deploy its Ubuntu Kubernetes cluster in under 2 weeks.</p>
+          <p class="u-hide--small"><a href="https://www.ubuntu.com/engage/kubernetes-case-study?utm_source=takeover&utm_medium=takeover&utm_campaign=2)FY19_Cloud_K8_CStudy_AllanGray" class="p-button--neutral u-no-margin--bottom" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Improving Productivity - Kubernetes takeover', 'eventLabel' : 'Download the case study', 'eventValue' : undefined });">Download the case study</a></p>
```
Comment (anthonydillon) [inline templates/takeovers/_improving_productivity_kubernetes.html]: Relative link please

File: templates/takeovers/_improving_productivity_kubernetes.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <div class="col-6 u-vertically-center">
+          <div class="u-align--center">
```
Comment (anthonydillon) [inline templates/takeovers/_improving_productivity_kubernetes.html]: Move `u-align--center` to the div above and remove this one please

File: templates/takeovers/_improving_productivity_kubernetes.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <img src="{{ ASSET_SERVER_URL }}1db850da-K8s+logo.svg" alt="Kubernetes logo" class="p-improv-k8-takeoever-image"/>
+          </div>
+          <p class="u-hide--large u-hide--medium"><a href="https://www.ubuntu.com/engage/kubernetes-case-study?utm_source=takeover&utm_medium=takeover&utm_campaign=2)FY19_Cloud_K8_CStudy_AllanGray" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Improving Productivity - Kubernetes takeover', 'eventLabel' : 'Download the case study', 'eventValue' : undefined });">Download the case study</a></p>
+        </div>
+      </div>
+  <style>
+    .p-improv-k8-takeoever-image{
```
Comment (anthonydillon) [inline templates/takeovers/_improving_productivity_kubernetes.html]: Please give takeovers the same block name. This should be `p-improving-prod-takeover__image`

File: templates/takeovers/_improving_productivity_kubernetes.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<section class="p-strip--image is-dark is-deep u-image-position js-takeover p-improving-prod-takeover">
+      <div class="row">
```
Comment (anthonydillon) [inline templates/takeovers/_improving_productivity_kubernetes.html]: This row will need `u-equal-height` to make vertical alignment work

File: templates/takeovers/_improving_productivity_kubernetes.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    
+    .p-improving-prod-takeover {
+      background: #173D8B;
+      background-image: linear-gradient(-74deg, #173D8B 8%, #326DE6 92%);
+    }
+
+    @media screen and (min-width: 874px) {
+      .p-improving-prod-takeover{ 
+        background-image: url("{{ ASSET_SERVER_URL }}0bfde879-suru-left-side.png"), url("{{ ASSET_SERVER_URL }}4c072e93-suru-right-side.png");
```
Comment (Caleb-Ellis) [inline templates/takeovers/_improving_productivity_kubernetes.html]: The linear gradient is getting overwritten here. Add it back to this background-image like so:
`url("{{ ASSET_SERVER_URL }}0bfde879-suru-left-side.png"), url("{{ ASSET_SERVER_URL }}4c072e93-suru-right-side.png"), linear-gradient(-74deg, #173D8B 8%, #326DE6 92%)`

File: templates/index.html [hunk 6]
```diff
@@ -24,5 +24,6 @@
 {% block takeover_content %}
   {% include "takeovers/_german_takeover_k8.html" %}
   {% include "takeovers/_german_takeover_openstack_made_easy.html" %}
-  {% include "takeovers/_1810-webinar-takeover.html" %}
+  <!-- {% include "takeovers/_1810-webinar-takeover.html" %} -->
```
Comment (anthonydillon) [inline templates/index.html]: Can you remove the comments here please

File: templates/takeovers/_improving_productivity_kubernetes.html [hunk 7]
```diff
@@ -0,0 +1,36 @@
+<section class="p-strip--image is-dark is-deep u-image-position js-takeover p-improving-prod-takeover">
+      <div class="row u-equal-height">
```
Comment (anthonydillon) [inline templates/takeovers/_improving_productivity_kubernetes.html]: Over indented

File: templates/takeovers/_improving_productivity_kubernetes.html (hunk 5 above)
Comment (inuwa) [reply templates/takeovers/_improving_productivity_kubernetes.html]: @Caleb-Ellis Thanks. Also we had to add to a background-blend-mode. Cheers.

File: templates/index.html (hunk 6 above)
Comment (inuwa) [reply templates/index.html]: Resolved

File: templates/takeovers/_improving_productivity_kubernetes.html (hunk 7 above)
Comment (inuwa) [reply templates/takeovers/_improving_productivity_kubernetes.html]: Removed the indentation.

**Comment by inuwa** [comment]
@anasereijo the changes have been made which you requested.

**Comment by anasereijo** [comment]
@inuwa please remove the suru background on iPad Pro as isn't resizing properly and then it's ready to go

**Comment by inuwa** [comment]
@anasereijo it is done now. Cheers.

**Comment by Khutch25** [comment]
Can the title/head be updated to "Boosting efficiency with Ubuntu Kubernetes" - It flows better into the description below and is shorter. @Greggless @shivjoshi1996

**Pull request merged**